### DISCUSSION TITLE
Error Using `latex-workshop.bibalign` Command in VSCode LaTeX Workshop

### DISCUSSION THREAD

**Discussion by Foadsf** [James-Yu/LaTeX-Workshop#4144] [Q&A]
I am encountering an issue with the LaTeX Workshop extension for Visual Studio Code. I'm attempting to format my `.bib` files using the `latex-workshop.bibalign` command, as per the guidelines provided in the [LaTeX Workshop wiki](https://github.com/James-Yu/LaTeX-Workshop/wiki/Format).

However, when I run the command, I receive the following error message:

> Command 'LaTeX Workshop: Align BibTeX file' resulted in an error<br/>Expected ... but '@' found.

[![enter image description here][1]][1]

The error suggests a syntax issue with the BibTeX file, but I am unsure what's causing it or how to fix it. The file contains regular BibTeX entries; I haven't made any unconventional modifications.

Has anyone encountered this error before, or does anyone know what might be causing it? Any insights or suggestions on how to resolve this would be greatly appreciated.

**Additional Info**: I'm using MiKTeX on Windows 10, and my `.bib` files are a mixture of BibTeX and BibLaTeX.

**P.S.** I have also posted this question [here](https://tex.stackexchange.com/q/707935/102825) on tex.stackexchange.com and [here](https://www.reddit.com/r/LaTeX/comments/19f8tgn/error_using_latexworkshopbibalign_command_in/?utm_source=share&utm_medium=web2x&context=3) on r/LaTeX subreddit.

  [1]: https://i.stack.imgur.com/hYRLP.png

**Comment by James-Yu** [comment]
Please file an issue and provide a minimal working (or rather, not working) example.

**Comment by Foadsf** [comment]
@James-Yu  The issues is that it works fine on the MWEs I have (e.g., [here](https://tex.meta.stackexchange.com/a/4408/102825) and [here](https://github.com/PHPirates/biber-biblatex-mwe/blob/master/src/references.bib)). You may see my complete `.bib` file attached.
[references.zip](https://github.com/James-Yu/LaTeX-Workshop/files/14052654/references.zip)
